### ISSUE TITLE
Issue with Steam again

### ISSUE THREAD

**Issue by amarildojr** [netblue30/firejail#2263]
I've been away from Linux for a year I think, and now that I'm back I'm again experiencing issues with Firejail and Steam. It seems every time I come back firejail conflicts with something, be it LibreOfice, Steam, VLC, or whatever :/ 

This time I don't really know what the issue is. Yesterday I installed Steam and ran it through Firejail, no problems at all.

Today after a reboot Steam won't open anymore. Nothing is shown in the terminal, as you can see here https://pastebin.com/raw/vHX0PVu1

The client actually opens, but after I type my credentials and click "Login" it closes.

I also forgot how to debug Steam with Firejail. But the command for debugging Steam is 'DEBUGGER=gdb steam'.

I created a new user and ran Steam without firejail and it opened/logged me in with no problems, so to me it's clear Firejail is to blame.

Any ideas? 
I'll edit Steam's profile and comment some lines to see which one(s) are the problem.

Info:
Debian Stable
Backported Kernel (4.18.X) and mesa and it's related stuff (18)
Firejail from any versions don't work (currently using the latest one from backports)
AMD R9 270X with OSS drivers (radeon Kernel driver)

**Comment by SkewedZeppelin** [comment]
Have you tried starting fresh?
`firejail steam --reset`

otherwise try starting with private-etc

**Comment by SkewedZeppelin** [comment]
Try manually then, be sure to back up and game files/saves/etc.
```
cd ~
rm -rf .steampid .steampath .steam .local/share/Steam
```

**Comment by amarildojr** [comment]
Thanks but I already did that.

**Comment by SkewedZeppelin** [comment]
There is always this
```
flatpak install flathub com.valvesoftware.Steam
flatpak --user override --nofilesystem=xdg-music com.valvesoftware.Steam
flatpak --user override --nofilesystem=xdg-pictures com.valvesoftware.Steam
flatpak --user override --nodevice=all com.valvesoftware.Steam
flatpak --user override --device=dri com.valvesoftware.Steam 
```
:rofl:

**Comment by amarildojr** [comment]
Weird. Looking through the backtrack I found:
"abort () from /lib/i386-linux-gnu/libc.so.6"
After pasting it on UncleGoogle I found this thread (post #10): https://steamcommunity.com/app/221410/discussions/0/1750106661716029502/

Changed it to "en_US" and Steam is now working.

Sometimes I just cannot understand Linux.

Anyways. Sorry Netblue for blaming Firejail, but it usually does conflict with some programs I use. Still,  it's no reason to jump to conclusions.

I'm closing this thread as the issue is now solved.
